utils/staking-miner/src/signer.rs

```diff
@@ -69,6 +73,12 @@ pub(crate) async fn read_signer_uri<
 	let _info = get_account_info::<T>(&client, &account, None)
```

Comment:
why underscore?

Reply:
because it is only used in logging and if that log target is not enabled, this variable would not be used at all.